### PR TITLE
Create typography.css

### PR DESCRIPTION
**Pull request by Radheshyam1234**
Defined all the h1,h2,h3,h4,h5,h6,p, font-weight,text-alignment for all of my component  library

Typography-screenshot

### CODE REVIEW

File: Components/typography.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  }
+  h2 {
+    font-size: 1.75rem;
+  }
+  h3 {
+    font-size: 1.5rem;
+  }
+  h4,
+  h5,
+  h6 {
```
Comment (ZuberDunge) [inline Components/typography.css:103]: why same size for h4, h5, h6?

**Review by ZuberDunge — COMMENTED** [review]
looks good to me

**Review by junaidshaikh-js — COMMENTED** [review]
Looks good to me!

File: Components/typography.css (hunk 1 above)
Comment (Radheshyam1234) [reply Components/typography.css:103]: That is for the small screen

**Pull request merged**